src/chainparams.cpp

```diff
@@ -424,3 +434,69 @@ void SelectParams(const std::string& network)
     SelectBaseParams(network);
     pCurrentParams = &Params(network);
 }
+
+// bip135 begin
+/**
+ * Return true if a deployment is considered to be configured for the network.
+ * Deployments with a zero-length name, or a windowsize or threshold equal to
+ * zero are not considered to be configured, and will be reported as 'unknown'
+ * if signals are detected for them.
+ * Unconfigured deployments can be ignored to save processing time, e.g.
+ * in ComputeBlockVersion() when computing the default block version to emit.
+ */
+bool isConfiguredDeployment(const Consensus::Params& consensusParams, const int bit)
+{
+    const Consensus::BIP9Deployment *vdeployments = consensusParams.vDeployments;
+    const struct BIP9DeploymentInfo& vbinfo = VersionBitsDeploymentInfo[bit];
+
+    if (strlen(vbinfo.name) == 0)
+        return false;
+
+    if (vdeployments[bit].windowsize == 0 || vdeployments[bit].threshold == 0)
+    {
+        return false;
+    }
+    return true;
+}
+
+/**
+ * Return a string representing CSV-formatted deployments for the network.
+ * Only configured deployments satisfying isConfiguredDeployment() are included.
+ */
+const std::string NetworkDeploymentInfoCSV(const std::string& network)
```

Comment:
No, not purely debugging - the intended use cases:
- inspect the built-in configuration using the `-dumpforks` command line option (based on a review comment from BIP review that requested: "be self-documenting")
- redirect that to a file which can then be modified to easily create a custom configuration which can be loaded with the `-forks=csvfile` option